src/hybrid_reader.c

```diff
@@ -320,6 +320,8 @@ static int HR_ReadKnnUnsorted(void *ctx, RSIndexResult **hit) {
     return INDEXREAD_EOF;
   }
   hr->lastDocId = (*hit)->docId;
+  ResultMetrics_Reset(*hit);
+  ResultMetrics_Add(*hit, hr->base.ownKey, RS_NumVal((*hit)->num.value));
```

Comment:
How would I add it for GEO?
Not a blocker for this PR.

Reply:
1. First add a metric request on the node evaluation, make sure to specify the `ownKey` address.
2. use the `ResultMetrics_...` API to add, reset and all together modify the values